nyaa/templates/profile.html

```diff
@@ -14,6 +14,7 @@ <h2 style="margin-bottom: 20px;">Profile of <strong class="text-{{ g.user.userle
 			<dt class="col-sm-2">User ID:</dt><dd class="col-sm-10">{{ g.user.id }}</dd>
 			<dt class="col-sm-2">User Class:</dt><dd class="col-sm-10">{{ g.user.userlevel_str }}</dd>
 			<dt class="col-sm-2">User Created on:</dt><dd class="col-sm-10">{{ g.user.created_time }}</dd>
+            <dt class="col-sm-2"><a href="{{ url_for('users.view_own_comments') }}">View all comments</a></dt>
```

Comment:
tabs vs. spaces; HTML template files use tab for tab width adjustment convenience

Reply:
Oh, I didn't even notice that, sorry.